shared/scss/views/_privacy-practices.scss

```diff
@@ -0,0 +1,47 @@
+.privacy-practices {
+    min-height: 100%;
+    box-sizing: border-box;
```

Comment:
This made me realize that our normalize file is not making border-box the default. Maybe I should do that. Hm.